.travis.yml

```diff
@@ -8,3 +8,7 @@ before_script:
 test:
   adapter: postgresql
   database: vss_test
+env:
+  global:
+    - secure: EvFAo0/MjGS7oRGlUaLeskLWVrobizdYft0keN/QF8QO/9IvCPILoJhAxwHaBrQPZsrnkDNtRRxQRc+R3a2zpIxZPknYPQt/1xKm6cOIihfF5wDzxjSykHFwsVg3Rqoikx0ejDgz/yYA9eJIVYy9u/7tUN6fw523c+8a4m0DIMoToYzltv8skBHHOewjWielcFtFLti6rMZ7JUUTAfLqeMB++/kN5OH7VqkwDJP95bw/BFRAWLCM48VFcUxhEs2Nkf6NcvArR8Sv4garYEjn0oSlUYD9niBLbQ4tq4WQJQx6o+7VVDUuFFVYVuQgG34txFJaHtcoXDARNIk2zcAzzPHFIHz/uYi3HnaIR5PBJIn1k9+eRMsW426ywdbPeqAVFpW6JNPKINhekiqVFxsOL9p4BInHs7yseqlIfA04UtMxTsQqlwh+71bfSuK2XolUtjHwTTxl35XCytAcdd3sbBKuC8XsNbW29G3/12Hh7+ymzASyatgkWy+nQMoNXgav0fTlq1JYUgli0kPRY2EnBPtezqvWI4of+PHiMfzq3aoEHpNRKXxTYbmXvvT/sCO4SVHNf4KqhpQ2DuJ3E+v4x7VRdrtMpg57AZ+DFza+oVjrGIFbN0V0hnWACX9tJQCu3N66CpCCrS2qa/kUHrdjoh6g35xkPe33RdI1c3dxuak=
+    - secure: dgRyBsH374kZ4pWG8XLd5P6HSofba0oeCc2TVzvdK9vnCNMxIgNDzryVYx6NpJwiQWagLYIQPZXwvvc3xYdq2A1oDAN/rTUFJzplKNSwU/KORhwXiMFPJk7CDJabf5va5mQAJ+wu1ruBNJm/cCgYLhBVjWkHBTAGiLwvDr8Zpk5hzmfaRdNeLEDRlDKNn0mz2MUs6OdH94VtRhytdHCKtmoNXgk2n4mbw+LDGggeYu929o3RNeyMAYoOFDen4pOsvQJ1rBbUKmr9nlJ9sxMEGS2YMLejJxI9k4r5Ax9BeyRNS4m5dwxUsQHNcFQUq36JMpcnBsWls3TRW+ba7YxVSQOX9utJP0YiAg9wVsg7kxN34GAH22dMPYzS67k5LKZK7ztobehUtxigH0GFoiOfvS8ov7h6V5Zb2kxpXbG3NOAT2rDAKEg/c2tDkV2DwBBubC0rzHKnN6khQhDFpshrITQlpr4N0/aejayURSmNkNWyog2uZHzduvKIf9cTbsKqN0FgQJJB2QGYxue9DNS/EyIEJaVRkkJyKzjf7/FCzElSBHXXgXyYzQl/Gw936iZoenxRcVRQlSe+DzHwA3xoePFzWAdJeBmhdxLhCPcyhXE9hU5r+lvHrFjn5iSN3qqE8uzFO0zva+1q1G1kYxBDohx6p+OJYnje2D1YqZ4XoMg=
```

Comment:
why there are two?

Reply:
ah, I only need the test key... have amended.